library/spdm_requester_lib/libspdm_req_handle_error_response.c

```diff
@@ -103,6 +103,8 @@ static libspdm_return_t libspdm_requester_respond_if_ready(libspdm_context_t *sp
 libspdm_return_t libspdm_handle_simple_error_response(libspdm_context_t *spdm_context,
                                                       uint8_t error_code)
 {
+    spdm_set_certificate_request_t *last_spdm_request = (void *)spdm_context->last_spdm_request;
```

Comment:
we do not do assignment when define the variable.

Please split to two lines.

It had better assign the value just before use at line 116.